homeassistant/components/fronius/__init__.py

```diff
@@ -153,7 +153,7 @@ async def _create_solar_net_device(self) -> DeviceInfo:
         )
         if self.logger_coordinator:
             _logger_info = self.logger_coordinator.data[SOLAR_NET_ID_SYSTEM]
-            solar_net_device[ATTR_MODEL] = _logger_info["product_type"]["value"]
+            solar_net_device[ATTR_MODEL] = _logger_info.get("product_type", {"value": "fronius-datalogger-web"})["value"]
```

Comment:
Thanks very much!

Default DeviceInfo.manufacturer is `Fronius`. So I guess we can set the default for `model` to "Datalogger Web". No need for sluggified strings - this is for UI only.
```suggestion
            # API v0 doesn't provide product_type
            solar_net_device[ATTR_MODEL] = _logger_info.get("product_type", {}).get(
                "value", "Datalogger Web"
            )
```
or imho more readable
```suggestion
            try:
                solar_net_device[ATTR_MODEL] = _logger_info["product_type"]["value"]
            except KeyError:
                # API v0 doesn't provide product_type
                solar_net_device[ATTR_MODEL] = "Datalogger Web"
```
but this would require testing to keep coverage to 100%